src/constants/DefaultConstants.js

```diff
@@ -81,8 +82,45 @@ export const ACTION_STATUS = {
 
 export const ROLE = {
   ADMIN: "Admin",
-  DOCTOR: "Regular User",
+  DOCTOR: "Doctor",
```

Comment:
add comment that this is deprecated and will be replaced

Reply:
done